scripts/video-helper.js

```diff
@@ -155,6 +155,57 @@ export const createVideo = (src, className = '', props = {}) => {
     },
   });
 
+  // Playback controls - play and pause button
+  const playPauseButton = createElement('button', {
+    props: { type: 'button', class: 'v2-video__playback-button' },
+  });
+
+  const pauseIconFragment = document.createRange().createContextualFragment(`
```

Comment:
You can use a single fragment to create both controls

```
const videControls = document.createRange().createContextualFragment(`
    <span class="icon icon-pause-video">
       <svg width="27" height="27" viewBox="0 0 72 72" fill="none" xmlns="http://www.w3.org/2000/svg">
          <circle cx="36" cy="36" r="30" fill="white"/>
          <rect x="28.25" y="24.45" width="2.75" height="23.09" fill="#141414"/>
          <rect x="41" y="24.45" width="2.75" height="23.09" fill="#141414"/>
       </svg>
    </span>
    <span class="icon icon-play-video">
      <svg width="27" height="27" viewBox="0 0 72 72" fill="none" xmlns="http://www.w3.org/2000/svg">
        <circle cx="36" cy="36" r="30" fill="white"/>
        <path fill-rule="evenodd" clip-rule="evenodd" d="M49.3312 35.9998L29.3312 24.4528L29.3312 47.5468L49.3312 35.9998ZM44.3312 35.9998L31.8312 28.7829L31.8312 43.2167L44.3312 35.9998Z" fill="#141414"/>
      </svg>
    </span>`);

  playPauseButton.append(...videControls.children);
```

Reply:
Nice tip, thanks :)